src/commands/remoteDebugJava/remoteDebugJavaFunctionApp.ts

```diff
@@ -40,7 +40,7 @@ export async function remoteDebugJavaFunctionApp(context: IActionContext, node?:
                 const appSettings: WebSiteManagementModels.StringDictionary = await client.listApplicationSettings();
                 if (needUpdateSiteConfig(siteConfig) || (appSettings.properties && needUpdateAppSettings(appSettings.properties))) {
                     const confirmMsg: string = localize('confirmRemoteDebug', 'The configurations of the selected app will be changed before debugging. Would you like to continue?');
-                    const result: vscode.MessageItem = await context.ui.showWarningMessage(confirmMsg, { modal: true }, DialogResponses.yes, DialogResponses.learnMore, DialogResponses.cancel);
+                    const result: vscode.MessageItem = await context.ui.showWarningMessage(confirmMsg, { modal: true }, DialogResponses.yes, DialogResponses.learnMore);
```

Comment:
missing 'stepName'?

Reply:
👀  Yeah I got a bit lazy for scenarios that I don't really care about... and this is one of those scenarios. there's just so many prompts